src/ansible_dev_tools/resources/server/creator.py

```diff
@@ -67,9 +70,14 @@ def playbook(
             return result
         with tempfile.TemporaryDirectory() as tmp_dir:
             # result.body here is a dict, it appear the type hint is wrong
-            tar_file = CreatorBackend(Path(tmp_dir)).playbook(
-                **result.body,  # type: ignore[arg-type]
-            )
+            if "v1/creator/playbook" in request.path:
```

Comment:
This check can go away if we remove the `playbook_v2` function

Reply:
Modified this check to used `playbook_v1` function now.